mac/KeymanEngine4Mac/KeymanEngine4Mac/KME/OnScreenKeyboard/OSKView.m

```diff
@@ -17,6 +17,9 @@
 #include <Carbon/Carbon.h>
 #import "KMELogs.h"
 
+const int64_t OSK_EVENT_FLAG = 0x8000000000000000;
```

Comment:
How was this flag selected?

Reply:
I am copying the modifier flags into the lower 32 bits and saving the upper 32 bits to mark this as our custom user data. Since it defaults to zero otherwise, I am just setting the upper-most bit of the upper 32, and I can use the other 31 bits in the future if I need to pass any other info from the OSK through to the event tap.